netstandard/ref/netstandard.csproj

```diff
@@ -9,15 +9,17 @@
     <IsCoreAssembly>true</IsCoreAssembly>
     <!-- disable obsolete warnings/errors -->
     <NoWarn>$(NoWarn);0618;0619;0809</NoWarn>
-
     <DefaultBuildAllTarget>Build</DefaultBuildAllTarget>
     <BuildAllProjects>true</BuildAllProjects>
+    <SignType>oss</SignType>
```

Comment:
Chances are good that if it is wearing the OSS signtype, I added it because PRSS was refusing to sign it owing to a mismatch. 

I did this in a 'broad strokes' kind of manner, so I will see if this works without the OSS type, if not we can chat.